packages/core/core/src/applyRuntimes.js

```diff
@@ -41,6 +40,23 @@ type RuntimeConnection = {|
   isEntry: ?boolean,
 |};
 
+function nameRuntimeBundle(
+  bundle: InternalBundle,
+  siblingBundle: InternalBundle,
+) {
+  // We don't run custom namers on runtime bundles as the runtime assumes that they are
+  // located at the same nesting level as they're owning bundle. Custom naming could
+  // be added in future as long as the custom name is validated.
+  let {hashReference} = bundle;
+
+  let name = siblingBundle.name
+    .replace(`.${bundle.type}`, `.runtime.${hashReference}.${bundle.type}`)
+    .replace(siblingBundle.hashReference, 'sib-hash');
```

Comment:
Does `sib-hash` get replaced somewhere?

Reply:
This is just a placeholder implementation, we'll need to do something smarter here. Sorry if that wasn't clear.